scylla/templates/scylla-statefulset.yaml

```diff
@@ -58,9 +62,15 @@ spec:
             mountPath: /opt/ready-probe.sh
             subPath: ready-probe.sh
       volumes: # TODO: data is discarded if pod restarts, change to something more persistant 
```

Comment:
This comment only makes sense if there's no hostPath defined

Reply:
done